cmd/runtimetest/main.go

```diff
@@ -642,6 +644,12 @@ func validate(context *cli.Context) error {
 			err := v.test(spec)
 			t.Ok(err == nil, v.description)
 			if err != nil {
+				if logLevel >= logrus.ErrorLevel {
+					// Skip 'SHOULD', 'MAY' ...
```

Comment:
Yes, overloading logrus is a little bit tricky.
I'll use another flag like 'rfc-level'.